#! /bin/sh /usr/share/dpatch/dpatch-run
## 10-mythweb_directory.dpatch by laga <>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: This patch sets the mythweb directory in mythweb.conf.apache to 
## DP: "/var/www/mythweb"

@DPATCH@

--- a/mythweb/mythweb.conf.apache	2007-05-12 12:43:55.727415750 +0200
+++ b/mythweb/mythweb.conf.apache.ubuntu	2007-05-13 16:17:03.121961250 +0200
@@ -11,7 +11,7 @@
 #    /var/www/html/mythweb
 #    /srv/www/htdocs/mythweb
 #
-    <Directory "/var/www/html" >
+    <Directory "/var/www/mythweb/" >
 
     ############################################################################
     # I *strongly* urge you to turn on authentication for MythWeb.  It is disabled
